### DISCUSSION TITLE
Feature request: Don't load essential tabs on launch

### DISCUSSION THREAD

**Discussion by blakegearin** [zen-browser/desktop#2883] [Ideas]
It seems most of my essential tabs when loaded on launch add a notification/alert dot underneath. So I have to click on each to remove all the dots.

Also, even if this wasn't an issue, there are social tabs like Discord I don't want to auto-load because it will make me appear online when I haven't taken any explicit action to be signed in to the site.

**Comment by pressRtowin** [comment]
better yet, don't load _any_ tabs on launch, or when switching workspaces. I don't want any tabs opened until I explicitly click on one. This is how Arc behaves and it's preferable in just about every way.

**Reply by j-peeters** [reply]
I would also like this behaviour. Currently I always have to refresh my essentials and pinned tabs to see the latest information on a website, e.g. news sites. With Arc, you click, it loads and you see the latest version of a page. That would be preferable.
And thanks for the great work on Zen, dear devs. It is really appreciated. You are on a roll!

**Comment by Spimy** [comment]
This needs to happen because the first thing I do when I launch my browser now is unloading all the tabs which can get very annoying.
